saltstack/base/salt/prerequisites/pip.sls

```diff
@@ -30,3 +32,10 @@ pip_install_requests_security:
 pip_install_virtualenvwrapper:
   pip.installed:
     - name: virtualenvwrapper
+
+install_jq:
```

Comment:
is jq install required to move ?

Reply:
there was an issue with salt and python, so I had to move